### PR TITLE
[default-login] Using capital first letter to render SanityLogo

### PR DESCRIPTION
**Pull request by kristofferj**
Strange thing. React seems to not render functions when lowercase first letter.

### CODE REVIEW

File: packages/@sanity/default-login/src/CookieTest.js [hunk 1]
```diff
@@ -39,17 +39,22 @@ class CookieTest extends PureComponent {
...
-
-          <div className={styles.sanityLogo}>
-            {sanityLogo}
-          </div>
+          { SanityLogo && (
+            <div className={styles.sanityLogo}>
+              <SanityLogo />
+            </div>
+          )}
+          { sanityLogo && (
```
Comment (skogsmaskin) [inline packages/@sanity/default-login/src/CookieTest.js]: sanityLogo && !SanityLogo perhaps?

File: packages/@sanity/default-login/src/LoginDialog.js [hunk 2]
```diff
@@ -71,14 +70,20 @@ export default class LoginDialog extends React.Component {
...
     return (
       <div className={styles.root}>
 
         <div className={styles.inner}>
+          { SanityLogo && (
+            <div className={styles.sanityLogo}>
+              <SanityLogo />
+            </div>
+          )}
           { sanityLogo && (
```
Comment (skogsmaskin) [inline packages/@sanity/default-login/src/LoginDialog.js]: sanityLogo && !SanityLogo perhaps here too.

File: packages/@sanity/default-login/src/LoginWrapper.js [hunk 3]
```diff
@@ -24,11 +24,11 @@ export default class LoginWrapper extends React.PureComponent {
     ]).isRequired,
     title: PropTypes.node,
     description: PropTypes.node,
-    sanityLogo: PropTypes.oneOfType([
+    SanityLogo: PropTypes.oneOfType([
       PropTypes.node,
       PropTypes.func
     ]),
```
Comment (skogsmaskin) [inline packages/@sanity/default-login/src/LoginWrapper.js]: I think the backward-compatibility is missing here?

Comment (skogsmaskin) [reply packages/@sanity/default-login/src/LoginWrapper.js]: Also, maybe make a console.warn when sanityLogo is provided mentioning that the prop i deprecated?

File: packages/@sanity/default-login/src/LoginWrapper.js [hunk 4]
```diff
@@ -79,7 +81,7 @@ export default class LoginWrapper extends React.PureComponent {
           <LoginDialog
             title={this.props.title}
             description={this.props.description}
-            sanityLogo={sanityLogo}
```
Comment (skogsmaskin) [inline packages/@sanity/default-login/src/LoginWrapper.js]: Shouldn't this one be:
```
SanityLogo={SanityLogo}
sanityLogo={sanityLogo}
```

File: packages/@sanity/default-login/src/LoginWrapper.js [hunk 5]
```diff
@@ -24,11 +24,12 @@ export default class LoginWrapper extends React.PureComponent {
     ]).isRequired,
     title: PropTypes.node,
     description: PropTypes.node,
-    sanityLogo: PropTypes.oneOfType([
+    sanityLogo: PropTypes.node,
+    SanityLogo: PropTypes.oneOfType([
       PropTypes.node,
```
Comment (skogsmaskin) [inline packages/@sanity/default-login/src/LoginWrapper.js]: The props for SanityLogo should now be PropTypes.func right?

File: packages/@sanity/default-login/src/LoginWrapper.js (hunk 4 above)
Comment (kristofferj) [reply packages/@sanity/default-login/src/LoginWrapper.js]: In loginwrapper we have defaultprop on SanityLogo (SanityStudioLogo), so if you send in sanityLogo it should override.

File: packages/@sanity/default-login/src/LoginWrapper.js (hunk 5 above)
Comment (kristofferj) [reply packages/@sanity/default-login/src/LoginWrapper.js]: yes

**Pull request merged**